lib/iris/coords.py

```diff
@@ -85,6 +86,8 @@ def _sort_key(defn):
                     defn.long_name is not None, defn.long_name,
                     defn.var_name is not None, defn.var_name,
                     defn.units is not None, defn.units,
+                    # Should attributes be included here?
```

Comment:
@trexfeathers Rather than have a comment here, can you make an associated issue? Thanks